wp-content/themes/cambodia-ict-camp/template-highlight.php

```diff
@@ -87,7 +87,10 @@
                                     <div class="col-xs-12 col-sm-4 col-md-4">
                                         <?php echo $responsive_profile; ?>
                                         <div class="text-center margin-bottom-10-px">
-                                            <h3><?php the_content(); ?></h3>
+                                            <h3 class="inline">
+                                                <p><a href=""><i class="fa fa-download"></i></a></p>
```

Comment:
What's happen to remove the P tag? 
If you  would like to show the icon inline, so you should not use P tag, but div or span can use.  I can't test, so I can't see how it will look if P is removed.